network/network_windows.go

```diff
@@ -318,6 +325,45 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *NetworkInfo, extIf *extern
 	return hcnNetwork, nil
 }
 
+func (nm *networkManager) addIPv6DefaultRoute() error {
+	// add ipv6 default route if it does not exist in dualstack overlay windows node from persistentstore
+	getIpv6IfIndexCmd := `((Get-NetIPInterface | where InterfaceAlias  -Like "vEthernet*").IfIndex)[0]`
+	ifIndex, err := nm.plClient.ExecutePowershellCommand(getIpv6IfIndexCmd)
+	if err != nil {
+		return errors.Wrap(err, "error while executing powershell command to get ipv6 Hyper-V interface")
+	}
```

Comment:
can you add a comment saying "getting interface index to add ipv6 default route"?

@princepereira / @paulyufan2  can there be multiple `vEthernet*` interfaces? If so, how cni know which is the right interface?

Reply:
as discussed, I will use (Get-NetIPAddress | where IPAddress -EQ "IPV6 Management IP").InterfaceIndex to get the interfaceIndex

Reply:
Or we know only have one Ethernet interface if we do not consider swiftv2 as multiple interfaces; so above command is working; otherwise we have to use above command and add ipv6 management IP from the network;

Reply:
as discussed we only consider single NIC scenario for now and so we can use this command: `((Get-NetIPInterface | where InterfaceAlias  -Like "vEthernet*").IfIndex)[0]`